src/core/public/application/ui/app_container.tsx

```diff
@@ -107,12 +108,14 @@ export const AppContainer: FunctionComponent<Props> = ({
   return (
     <Fragment>
       {appNotFound && <AppNotFound />}
-      {showSpinner && (
-        <div className="appContainer__loading">
-          <EuiLoadingElastic aria-label="Loading application" size="xxl" />
-        </div>
+      {showSpinner && !appNotFound && (
+        <EuiLoadingElastic
+          className="appContainer__loading"
+          aria-label="Loading application"
```

Comment:
i18n please

Reply:
If you can help me fix the tests, I can update the i18n 😉